beacon-chain/p2p/discovery.go

```diff
@@ -398,6 +508,19 @@ func (s *Service) isPeerAtLimit(inbound bool) bool {
 	return activePeers >= maxPeers || numOfConns >= maxPeers
 }
 
+func (s *Service) isBelowThreshold() bool {
+	maxPeers := int(s.cfg.MaxPeers)
```

Comment:
Here we cast `uint` to `int`.
Maybe it would be safer to do the opposite?

Reply:
The main reason I did it this way is because some of the other methods that we call all return `int` instead of `uint`. We would have to cast those to `uint` too in this particular case. Since this is a user defined value I think it is fine to expect it to be a manageable value.